src/components/shared/Svg.css

```diff
@@ -75,6 +75,10 @@ html .arrow.expanded svg {
   transform: rotate(0deg);
 }
 
+.domain.debuggee {
+  fill: lightgreen;
```

Comment:
Note: Maybe there's a green defined in the themes we can use here @jasonLaster?

Reply:
var(--theme-toolbar-highlighted-color) ?

Reply:
@nt1m I tried your suggestion and here is a screenshot of what I got:

![screenshot](https://user-images.githubusercontent.com/31850821/31919484-c1778416-b830-11e7-9fa1-1567c3165ec5.JPG)

To tell you the truth, I like green color more.